From d419278962e7e6465af130b05f1669573a5d6b05 Mon Sep 17 00:00:00 2001
From: Tao Ren <taoren@meta.com>
Date: Fri, 2 Feb 2024 16:57:17 -0800
Subject: [PATCH 2/7] ARM: dts: aspeed: elbert: Enable eMMC device

Enable eMMC device in Elbert dts because the eMMC entries were removed
from ast2600-facebook-netbmc-common.dtsi.
---
 .../boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts   | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts
index 74f3c67e0eff..ff1009ea1c49 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts
@@ -201,3 +201,15 @@ fixed-link {
 		full-duplex;
 	};
 };
+
+&emmc_controller {
+	status = "okay";
+};
+
+&emmc {
+	status = "okay";
+
+	non-removable;
+	max-frequency = <25000000>;
+	bus-width = <4>;
+};
-- 
2.44.2

